packages/website/ts/components/portal/portal.tsx

```diff
@@ -340,62 +340,77 @@ export class Portal extends React.Component<PortalProps, PortalState> {
         );
     }
     private _renderWallet(): React.ReactNode {
+        const startOnboarding = this._renderStartOnboarding();
+        const isMobile = this.props.screenWidth === ScreenWidths.Sm;
```

Comment:
should we pull this logic out into `utils` ?

Reply:
I feel like simple 1 line, 1 expression boolean checks aren't worth it? I'm also realizing I probably want to be checking against 'Md'